src/main/java/org/example/homework7/EmailValidator.java

```diff
@@ -0,0 +1,22 @@
+package org.example.homework7;
+
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class EmailValidator {
+    public static void main(String[] args) {
+        emailValidator();
+    }
+
+    public static void emailValidator() {
```

Comment:
It is better to have method `isValidEmail(String email)` and call it from the main

Reply:
Improved